Though you might want to fix this part
```
 ./cogs/counting.py:300:31: E999 SyntaxError: unterminated string literal (detected at line 300)
            "cosh": math.cosh",
                              ^
1     E999 SyntaxError: unterminated string literal (detected at line 300)
```
Or I can do it after merging, your call.

--- a/cogs/counting.py
+++ b/cogs/counting.py
@@ -297,7 +297,7 @@ class Counting(commands.Cog):
             "degrees": math.degrees,
             "radians": math.radians,
             "sinh": math.sinh,
-            "cosh": math.cosh",
+            "cosh": math.cosh,
             "tanh": math.tanh,
             "asinh": math.asinh,
             "acosh": math.acosh,

